### PR TITLE
Fixed minimal quantity check in cart

### PR DESCRIPTION
**Pull request by thierrymarianne**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | The minimal product quantity in cart is not always applied |
| Type? | bug fix |
| Category? | FO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-1640 |
| How to test? | Update the minimal quantity of a product and check it is properly applied in cart |

### CODE REVIEW

File: controllers/front/CartController.php [hunk 1]
```diff
@@ -100,16 +100,23 @@ public function displayAjaxUpdate()
...
+        list(, $updatedProduct) = each($updatedProducts);
+        $productQuantity = $updatedProduct['quantity'];
+
         if (!$this->errors) {
             $this->ajaxDie(Tools::jsonEncode([
                 'success' => true,
                 'id_product' => $this->id_product,
-                'id_product_attribute' => $this->id_product_attribute
+                'id_product_attribute' => $this->id_product_attribute,
+                'quantity' => $productQuantity
```
Comment (maximebiloe) [inline controllers/front/CartController.php]: Missing the last comma

File: controllers/front/CartController.php [hunk 2]
```diff
@@ -100,16 +100,23 @@ public function displayAjaxUpdate()
...
             ]));
         } else {
             $this->ajaxDie(Tools::jsonEncode([
                 'hasError' => true,
                 'errors' => $this->errors,
+                'quantity' => $productQuantity
```
Comment (maximebiloe) [inline controllers/front/CartController.php]: Same here

**Review by maximebiloe — CHANGES_REQUESTED** [review]

File: controllers/front/CartController.php (hunk 1 above)
Comment (thierrymarianne) [inline controllers/front/CartController.php]: Fixed. Thanks.

File: controllers/front/CartController.php (hunk 2 above)
Comment (thierrymarianne) [inline controllers/front/CartController.php]: Fixed

**Comment by dSkrbic** [comment]
Q&A test: 
Not fully correct, see added video in LaForge BOOM-1306
Suppressing Q&A approbation label

**Comment by aleeks** [comment]
Maxime's requests are ok!

Thank you @thierrymarianne !

**Pull request merged**